kubernetes/ansible/roles/sunbird-monitoring/templates/alertrules.yaml

```diff
@@ -87,3 +87,89 @@ redis_process_names:
 
 service_health_checks:
   {{ service_health_checks | to_yaml | indent( width=2) }}
+
+api_response_upward_trend_threshold: "{{ api_response_upward_trend_threshold }}"
```

Comment:
Can you please add description for both variables?

Reply:
description added.